bundle/Command/LegacyInitCommand.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * File containing the LegacySrcSymlinkCommand class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishLegacyBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+
+class LegacyInitCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('ezpublish:legacy:init')
+            ->setDefinition(
+                array(
```

Comment:
Are we using old school array syntax here?

Reply:
php-cs-fixer is not installed on this repo, it can be done and we can then cleanup CS across supported branches as followup to this.